content/lang/header.en_template.php

```diff
@@ -64,5 +64,6 @@
 $LANG['NAV_IMG_LIB'] = 'Image Library';
 $LANG['NAV_IMG_CONTR'] = 'Image Contributors';
 $LANG['NAV_IMG_SEARCH'] = 'Image Search';
+$LANG['BREADCRUMB_THUMB_BLDR'] = 'Thumbnail Builder';
```

Comment:
I think for breadcrumb translations that are unlikely to appear on many pages, it might be best to put them in content/lang/imagelib/admin/thumbnailbuilder.[en, es, fr].php instead of here.